packages/sdk-js/jsr.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@growthbook/growthbook",
+  "version": "0.1.3",
```

Comment:
Our latest SDK version is `1.2.0`.  Does this need to match here?

Reply:
Do you think they should match? I wasn't sure either way, because the JSR is a different release process and non-compiled code. But I could see a case for matching, too